package-lock.json

```diff
@@ -1,4 +1,5 @@
 {
+  "requires": true,
```

Comment:
What does this change relate to?

Reply:
Is likely since your team members are running different versions of Node.js and therefore npm.

We have solved this by encouraging the use of 'nvm': https://github.com/alphagov/govuk-frontend/pull/462

In this way you have a .nvmrc file in your root that indicates the version everyone should use, and then you have the same npm version.

Reply:
Add `requires` field in `package-lock.json`

This is automatically added when running `npm istall` with newer
versions of npm (since 5.1.0).

More info: stackoverflow.com/a/45180638
Related npm PR: npm/npm#17508
Related npm Changelog: npm/npm@v5.1.0 (release)

Reply:
Ok, I've just noticed that this is described by the first commit as well, maybe I missed that when looking at the diff yesterday...